src/qt/intro.cpp

```diff
@@ -138,21 +135,20 @@ Intro::Intro(QWidget *parent, uint64_t blockchain_size, uint64_t chain_state_siz
     }
     const int prune_target_gb = PruneMiBtoGB(prune_target_mib);
     ui->prune->setText(tr("Discard blocks after verification, except most recent %1 GB (prune)").arg(prune_target_gb ? prune_target_gb : DEFAULT_PRUNE_TARGET_GB));
-    requiredSpace = m_blockchain_size;
```

Comment:
In commit "refactor: Replace static variable with data member" (02c6060ba65c160dcd336b57752ce304ecd4ca9f)

This line doesn't seem to be replaced by anything. It looks like `m_required_space_gb` will now be set to 0 instead of `m_blockchain_size` when `prune_target_gb` is greater than `m_blockchain_size_gb`.

It seems like this should set `m_required_space_gb = m_blockchain_size_gb + m_chain_state_size_gb` when `prune_target_gb > m_blockchain_size_gb`

Reply:
Yes, this is a bug, and it is easy to reproduce it. Going to fix it.